### PR TITLE
RF | CreateNewViewWithSelectLabelMyView

### PR DESCRIPTION
**Pull request by olpolezhaeva**
Delete fillNameField, isNameExist methodes.
Added AfterMethod.

### CODE REVIEW

**Comment by olpolezhaeva** [comment]
[RF | CreateNewViewWithSelectLabelMyView](https://trello.com/c/Id2EmsV0/527-rf-createnewviewwithselectlabelmyview)

File: src/test/java/CreateNewViewWithSelectLabelMyView.java [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-            nameOfView += String.valueOf((int) (Math.random() * 999));
-        }
-        fillNameField();
-        getDriver().findElement(By.xpath("//input[@name='Submit']")).click();
+        getDriver().findElement(By.id("ok")).click();
 
         Assert.assertEquals(nameOfView,
                 getDriver().findElement(By.xpath("//ul[@id='breadcrumbs']//a[contains(@href, '/view')]"))
                         .getText());
     }
```
Comment (anonymoustuna) [inline src/test/java/CreateNewViewWithSelectLabelMyView.java]: You might add space here.

File: src/test/java/CreateNewViewWithSelectLabelMyViewTest.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import org.testng.annotations.Test;
 import runner.BaseTest;
 
-public class CreateNewViewWithSelectLabelMyView extends BaseTest {
+public class CreateNewViewWithSelectLabelMyViewTest extends BaseTest {
 
     String nameOfView = "My new view";
```
Comment (anonymoustuna) [inline src/test/java/CreateNewViewWithSelectLabelMyViewTest.java:9]: You might add space here.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/CreateNewViewWithSelectLabelMyViewTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class CreateNewViewWithSelectLabelMyViewTest extends BaseTest {
+
+    String nameOfView = "My new view";
```
Comment (zenicko) [inline src/test/java/CreateNewViewWithSelectLabelMyViewTest.java:9]: Are you changing the field "nameOfVie"? - final

File: src/test/java/CreateNewViewWithSelectLabelMyViewTest.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class CreateNewViewWithSelectLabelMyViewTest extends BaseTest {
+
```
Comment (zenicko) [inline src/test/java/CreateNewViewWithSelectLabelMyViewTest.java:8]: The empty line

**Review by zenicko — APPROVED** [review]
Errors should never pass silently.

**Pull request merged**